### ISSUE TITLE
500 error when trying to group issues by status in plain view mode.

### ISSUE THREAD

**Issue by hbfontana** [Loriowar/redmine_issues_tree#32]
Hi there! Your plugin is being really useful for us here! Thank you!

On the other hand, we started to notice  an error when trying to use the "Group By" dropdown after installing the plugin. The error occurs only when we select "Status" as the criteria for grouping tasks/issues.  The message displayed on the screen is: "An error occurred while executing the query and has been logged. Please report this error to your Redmine administrator."

**The error logged in production.log file is the following :** 

> Started GET "/redmine/projects/lab-ld/issues" for 10.1.210.2 at 2016-05-25 14:51:49 -0300
> Processing by IssuesController#index as HTML
>   Parameters: {"project_id"=>"lab-ld"}
>   Current user: admin (id=1)
>   Rendered plugins/redmine_issues_tree/app/views/issues_trees/_issues_index_header_tags.html.haml (1.2ms)
>   Rendered plugins/redmine_issues_tree/app/views/issues_trees/_link_to_tree_view.html.haml (1.4ms)
>   Rendered queries/_filters.html.erb (249.6ms)
>   Rendered queries/_columns.html.erb (3.7ms)
>   Rendered issues/_list.html.erb (46.4ms)
>   Rendered issues/_sidebar.html.erb (11.3ms)
>   Rendered plugins/redmine_issues_tree/app/views/issues/index.html.erb within layouts/base (331.1ms)
>   Rendered plugins/computed_custom_field/app/views/hooks/_base_head.html.erb (0.4ms)
>   Rendered plugins/progressive_projects_list/app/views/application/_progressive_sidebar.html.erb (0.1ms)
>   Rendered plugins/progressive_projects_list/app/views/application/_progressive_recent_projects.html.erb (0.3ms)
> Completed 200 OK in 510ms (Views: 356.1ms | ActiveRecord: 34.0ms)
> Started GET "/redmine/projects/lab-ld/issues?utf8=%E2%9C%93&set_filter=1&f%5B%5D=&c%5B%5D=tracker&c%5B%5D=subject&c%5B%5D=assigned_to&c%5B%5D=status&c%5B%5D=$
> Processing by IssuesController#index as HTML
>   Parameters: {"utf8"=>"✓", "set_filter"=>"1", "f"=>[""], "c"=>["tracker", "subject", "assigned_to", "status", "done_ratio", "due_date"], "group_by"=>"status$
>   Current user: admin (id=1)
> Query::StatementInvalid: Mysql2::Error: Unknown column 'FALSE' in 'order clause': SELECT  `issues`.`id` AS t0_r0, `issues`.`tracker_id` AS t0_r1, `issues`.`p$
>   Rendered common/error.html.erb within layouts/base (0.3ms)
>   Rendered plugins/computed_custom_field/app/views/hooks/_base_head.html.erb (0.3ms)
>   Rendered plugins/progressive_projects_list/app/views/application/_progressive_sidebar.html.erb (0.1ms)
>   Rendered plugins/progressive_projects_list/app/views/application/_progressive_recent_projects.html.erb (0.2ms)
> Completed 500 Internal Server Error in 83ms (Views: 31.2ms | ActiveRecord: 11.3ms)

As you can see, we have other plugins installed, but we started no notice this after installing "Redmine Issues Tree" plugin.

**More information:**

> Environment:
>   Redmine version                3.1.4.stable
>   Ruby version                   2.0.0-p598 (2014-11-13) [x86_64-linux]
>   Rails version                  4.2.5.2
>   Environment                    production
>   Database adapter               Mysql2
> SCM:
>   Subversion                     1.7.14
>   Git                            1.8.3.1
>   Filesystem                     
> Redmine plugins:
>   computed_custom_field          0.0.6
>   progressive_projects_list      2.0.1
>   redmine_checklists             3.1.3
>   redmine_dashboard              2.7.0
>   redmine_issue_completion       0.0.2
>   redmine_issues_tree            0.0.7
>   redmine_mentions               0.0.1
>   redmine_my_page_queries        2.1.6
>   subtask_columns                0.0.3

We hope you have some time to take a look on this. Sorry for my english. Thanks in advance!

**Comment by Loriowar** [comment]
Hi! First of all, are you use branch of issues_tree plugin with name corresponding to your Redmine version? I.e. are you install plugin from branch `3.1.0` for your Redmine `3.1.4`? If no - most likely this is a source of the problem. If yes - let me know, I try to find incompatibility with a plugins from your list.

**Comment by hbfontana** [comment]
Hi there! I am using the version 3.1.0 of the plugin. (redmine_issues_tree-3.1.0.zip). First time I tried to install it, I used a newer version (master at that time) and I was not able to use it. Going trough the issues page, I saw that I should use the corresponding version and that´s what I did, removing the one installed and installing the version. 3.1.0. After that, I started to use the functionalities normally, except for the error I described.

**The funny thing was that the error stopped to happen since last week. Maybe as a result of restarting the server since I might have failed doing that after installing some other plugin.** Well, I will let you know if the error appears again and, in that case, I will try to find out more details about the conditions that could be generating this before contacting you.

Finally, thanks again for this awesome plugin!
